docs/pages/server-access/reference/cli.mdx

```diff
@@ -0,0 +1,15 @@
+
+
+(!docs/pages/includes/reference/cli/tctl-nodes.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-ls.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-ssh.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-config.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-proxy-ssh.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-recordings-ls.mdx!)
+
+(!docs/pages/includes/reference/cli/tsh-play.mdx!)
```

Comment:
I think we're overusing includes here. This makes it very hard to ensure that the page has a reasonable flow and transitions nicely from one section to the next.

I would use includes for smaller snippets that make sense to be reused in different places, not to write entire docs pages.